notification/docs/IntegrationGuide.md

```diff
@@ -0,0 +1,65 @@
+# Integration of payment into checkout process
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Contents**
+
+- [Deployment](#deployment)
+  - [Glossary](#glossary)
+  - [Requirements](#requirements)
+- [Configuration](#configuration)
+      - [Environment variables to configure the notification module:](#environment-variables-to-configure-the-notification-module)
+  - [After deployment](#after-deployment)
+    - [Register the endpoint](#register-the-endpoint)
+- [Note](#note)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Deployment
+
+### Glossary
+In this process, there are 2 parties involved:
+
+**Adyen** - the payment provider which will send notifications
```

Comment:
maybe we should use bullet points in here.

Reply:
New line is missing.

Reply:
https://github.com/commercetools/commercetools-adyen-integration/commit/2264e922f43e7aaa6ec444599abe92da6ead7a3d